utoipauto-core/src/file_utils.rs

```diff
@@ -110,95 +115,139 @@ fn find_segment_and_skip<'a>(segments: &'a [&str], to_find: &[&str], to_skip: us
 
 #[cfg(test)]
 mod tests {
+    use quote::ToTokens;
+
     use super::*;
 
     #[test]
     fn test_extract_module_name_from_path() {
         assert_eq!(
-            extract_module_name_from_path("./utoipa-auto-macro/tests/controllers/controller1.rs", "crate"),
+            extract_module_name_from_path("./utoipa-auto-macro/tests/controllers/controller1.rs", "crate")
+                .to_token_stream()
+                .to_string()
+                .replace(" ", ""),
```

Comment:
The `replace` here is to adjust for additional whitespace between tokens when converting with `to_string`, which happened for me in the tests even when testing with `cargo` only.